sdk/liftrqumulo/Azure.ResourceManager.LiftrQumulo/api/Azure.ResourceManager.LiftrQumulo.netstandard2.0.cs

```diff
@@ -0,0 +1,480 @@
+        public virtual Azure.AsyncPageable<Azure.ResourceManager.LiftrQumulo.JobRunResource> GetAllAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.ResourceManager.LiftrQumulo.JobRunResource>> GetAsync(string jobRunName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        System.Collections.Generic.IAsyncEnumerator<Azure.ResourceManager.LiftrQumulo.JobRunResource> System.Collections.Generic.IAsyncEnumerable<Azure.ResourceManager.LiftrQumulo.JobRunResource>.GetAsyncEnumerator(System.Threading.CancellationToken cancellationToken) { throw null; }
+        System.Collections.Generic.IEnumerator<Azure.ResourceManager.LiftrQumulo.JobRunResource> System.Collections.Generic.IEnumerable<Azure.ResourceManager.LiftrQumulo.JobRunResource>.GetEnumerator() { throw null; }
+        System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
+    }
+    public partial class JobRunData : Azure.ResourceManager.Models.ResourceData
+    {
+        public JobRunData() { }
+        public string AgentName { get { throw null; } }
+        public string AgentResourceId { get { throw null; } }
+        public long? BytesExcluded { get { throw null; } }
+        public long? BytesFailed { get { throw null; } }
+        public long? BytesNoTransferNeeded { get { throw null; } }
+        public long? BytesScanned { get { throw null; } }
+        public long? BytesTransferred { get { throw null; } }
+        public long? BytesUnsupported { get { throw null; } }
+        public Azure.ResourceManager.LiftrQumulo.Models.JobRunError Error { get { throw null; } }
+        public System.DateTimeOffset? ExecutionEndOn { get { throw null; } }
+        public System.DateTimeOffset? ExecutionStartOn { get { throw null; } }
+        public long? ItemsExcluded { get { throw null; } }
+        public long? ItemsFailed { get { throw null; } }
+        public long? ItemsNoTransferNeeded { get { throw null; } }
+        public long? ItemsScanned { get { throw null; } }
+        public long? ItemsTransferred { get { throw null; } }
+        public long? ItemsUnsupported { get { throw null; } }
+        public System.BinaryData JobDefinitionProperties { get { throw null; } }
+        public System.DateTimeOffset? LastStatusUpdate { get { throw null; } }
+        public Azure.ResourceManager.LiftrQumulo.Models.ProvisioningState? ProvisioningState { get { throw null; } }
+        public Azure.ResourceManager.LiftrQumulo.Models.JobRunScanStatus? ScanStatus { get { throw null; } }
+        public string SourceName { get { throw null; } }
+        public System.BinaryData SourceProperties { get { throw null; } }
+        public string SourceResourceId { get { throw null; } }
```

Comment:
Is this a `arm-id`? If yes please correct it by following https://github.com/Azure/autorest.csharp/blob/feature/v3/docs/mgmt/polishing.md#change-the-format-of-a-property